Cloud Application Manager/Administering Your Organization/user-authentication.md

```diff
@@ -7,9 +7,15 @@
 }}}
 
 ### Enable User Authentication
-In Cloud Application Manager enterprise organizations, users can sign in using any of the single sign-on authentication options you enable in the [admin console](admin-overview.md).
+In Cloud Application Manager enterprise organizations, users can sign in using any of the single sign-on authentication options
+
 ![admin-user1.png](../../images/cloud-application-manager/admin-user1.png)
 
+you can access to these settings in the Site Management of your account
```

Comment:
Change to:  "you can access these settings from the left side menu in the Management site of your organization"